### PR TITLE
fix(chat): disable possibility to edit message if message is streaming (Issue #808)

### PR DESCRIPTION
**Pull request by Alexander-Kezik**
**Description:**

Disable possibility to edit message if message is streamig.

Issues:

- https://github.com/epam/ai-dial-chat/issues/808

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by Alexander-Kezik** [comment]
/deploy-review
>- **Deployment failed**: [failed](https://gitlab.deltixhub.com/Clients/deltix/nightly-test/dial/review/-/pipelines/453671)

File: apps/chat/src/store/conversations/conversations.epics.ts [hunk 1]
```diff
@@ -1324,7 +1324,7 @@ const streamMessageEpic: AppEpic = (action$, state$) =>
         chatBody,
       };
     }),
-    mergeMap(({ payload, chatBody }) => {
```
Comment (mikitabut) [inline apps/chat/src/store/conversations/conversations.epics.ts]: Doing that most likely will break compare mode - please recheck

**Comment by mikitabut** [comment]
I think it's better and easier to not allow edit message if message is streaming or stop automatically generation for such cases

**Review by IlyaBondar — COMMENTED** [review]
@Alexander-Kezik 
Please, fix typo `mossaeg` in PR title and description

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-954.nightly-test.deltixhub.io
>E2E tests status: success

**Review by IlyaBondar — APPROVED** [review]

**Pull request merged**